api/controllers/ApplicationTypeController.js

```diff
@@ -296,16 +294,29 @@ module.exports = {
                                                     first_name: account.first_name,
                                                     last_name: account.last_name,
                                                     telephone: account.telephone,
-                                                  mobileNo: account.mobileNo,
-                                                  email: user.email,
+                                                    mobileNo: account.mobileNo,
+                                                    email: user.email,
                                                     confirm_email: user.email,
                                                     has_email: true
                                                 }).then(function(){
-                                                    if (req.session.appType ==2) {
-                                                        return res.redirect('/business-document-quantity?pk_campaign=Premium-Service&pk_kwd=Premium');
-                                                    }
-                                                    else if (req.session.appType ==3) {
-                                                        return res.redirect('/business-document-quantity?pk_campaign=DropOff-Service&pk_kwd=DropOff');
+                                                    if (
+                                                        req.session.appType == 2
+                                                    ) {
+                                                        return res.redirect(
+                                                            '/business-document-quantity?pk_campaign=Premium-Service&pk_kwd=Premium'
+                                                        );
+                                                    } else if (
+                                                        req.session.appType == 3
+                                                    ) {
+                                                        return res.redirect(
+                                                            '/business-document-quantity?pk_campaign=DropOff-Service&pk_kwd=DropOff'
+                                                        );
+                                                    } else if (
+                                                        req.session.appType === 4
+                                                    ) {
+                                                        return res.redirect(
+                                                            '/check-document-eligibility'
+                                                        );
```

Comment:
I find this pattern slightly cleaner than using multiple else/ifs or a switch statement. Especially since you're in callback hell and everything is so nested, it gets hard to read. This requires no ifs 🤓 

```suggestion
const appTypeRedirect = {
  2: res.redirect( '/business-document-quantity?pk_campaign=DropOff-Service&pk_kwd=DropOff' ),
  3: res.redirect( '/business-document-quantity?pk_campaign=DropOff-Service&pk_kwd=DropOff'),
  4: res.redirect('/check-document-eligibility' )
}
const redirect = appTypeRedirect[req.session.appType]

return redirect
```

Reply:
Totally agree, again I was just adding to what they had already but it makes sense to refactor it

- [ ] Use object here instead of long if/else